.github/workflows/update-nextcloud-ocp-matrix.yml

```diff
@@ -18,18 +21,18 @@ jobs:
       fail-fast: false
       matrix:
         branches: ['master']
-        target: ['stable26']
+        target: ['stable27']
```

Comment:
Revert

Reply:
Or simply bump the requirement. 27 is also EOL end of this month